website/docs/r/app.html.markdown

```diff
@@ -1,59 +1,57 @@
-* `app_path` - (Required, string) Define the path to the zip file of the application. The zip must contain all the application files directly within it and not inside a top-level folder. Typically, you should go to the directory where your application files reside and issue `zip -r myapplication.zip *`.
-* `app_version`	 - (Optional, string) Version of the application. If the application content in the file specified by _app_path_ changes, Terraform can't detect it. You can either change the application zip file name to let Terraform know that your zip content has changed, or you can use this attribute to let the provider know that the content changed without changing the _app_path_.
-* `tags` - (Optional, array of strings) Set tags on the application instance.
-
-**NOTE**: `Tags` are managed locally and not stored on the IBM Cloud service endpoint at this moment.
-
-## Attributes Reference
-
-The following attributes are exported:
-
-* `id` - The ID of the application.
+---
+layout: "ibm"
+page_title: "IBM: app"
+sidebar_current: "docs-ibm-resource-app"
+description: |-
+  Manages IBM Application.
+---
+
+# ibm\_app
+
+Provides an application resource. This allows applications to be created, updated, and deleted.
+
+## Example Usage
+
+```hcl
+data "ibm_space" "space" {
+  org   = "example.com"
+  space = "dev"
+}
+
+resource "ibm_app" "app" {
+  name                 = "my-app"
+  space_guid           = "${data.ibm_space.space.id}"
+  app_path             = "hello.zip"
+  wait_timeout_minutes = 90
+  buildpack            = "sdk-for-nodejs"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required, string) The name of the application. You can retrieve the value by running the `bx app list` command in the [Bluemix CLI](https://console.ng.bluemix.net/docs/cli/reference/bluemix_cli/index.html#getting-started).
+* `memory` - (Optional, integer) The amount of memory, specified in megabytes, that each instance has. If you don't specify a value, the system assigns pre-defined values based on the quota allocated to the application. You can check the default values by running `bx cf org <org-name>`. The command lists the quotas that are defined in your organization and space. If space quotas are defined, you can get them by running `bx cf space-quota <space-quota-name>`, where <quota-name> is the name of the quota. Otherwise you can check the organization quotas by running `bx cf quota <quota-name>`.
+* `instances` - (Optional, integer) The number of instances of the application.
+* `disk_quota` - (Optional, integer) The maximum amount of disk, specified in megabytes, available to an instance of an application. The default value is [1024 MB](http://bosh.io/jobs/cloud_controller_ng?source=github.com/cloudfoundry/cf-release&version=234#p=cc.default_app_disk_in_mb). Check with your cloud provider if the value has been set differently.

```

Comment:
What does the sentence "Check with your cloud provider if the value has been set differently." mean? How would users know or check if the value has been set differently? @vkalangu